styles/vision_style.css

```diff
@@ -0,0 +1,10 @@
+.circulo {
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+}
+
+.centro {
+    display: flex;
+    justify-content: center;
+}
```

Comment:
add the header and footer style in the home_style.css file

Reply:
done